mypy/newsemanal/semanal_main.py

```diff
@@ -278,6 +279,9 @@ def semantic_analyze_target(target: str,
             analyzer.refresh_partial(refresh_node, patches, final_iteration)
             if isinstance(node, Decorator):
                 infer_decorator_signature_if_simple(node, analyzer)
+    for dep in analyzer.imports:
+        state.dependencies.append(dep)
+        state.priorities[dep] = mypy.build.PRI_LOW
```

Comment:
There is no checking against what is already a dependency, so can this result in things ending up in the dependency list multiple times or having the priority clobbered?
(Though I suppose we hope that the priority does not matter with the new semantic analyzer, right?)